.travis.yml

```diff
@@ -1,31 +1,44 @@
 ---
 sudo: false
+dist: trusty
 language: ruby
 cache: bundler
-script: "bundle exec rake release_checks"
-#Inserting below due to the following issue: https://github.com/travis-ci/travis-ci/issues/3531#issuecomment-88311203
 before_install:
-  - gem update bundler
+  - bundle -v
+  - rm -f Gemfile.lock
+  - gem update --system
+  - gem --version
+  - bundle -v
```

Comment:
I'd suggest either moving these to a helper script,

```yaml
before_install: some/where/prep-tests
```

 or using a different approach, i.e. say with one of the yaml multi-line sytaxes.

For example, using ">":

```yaml
before_install: >
    bundle -v 
    && rm -f Gemfile.lock
    && ...
```
or using "|"

```yaml
before_install: |
    set -e
    bundle -v 
    rm -f Gemfile.lock
    ...
```
(Note that syntax is just approximate -- haven't checked it.)

Otherwise, travis will just keep going iirc.  That is, it runs all the bullet points, even if one of them fails, only stopping if there was a failure after it has tried them all.